semgrep-core/src/parsing/Parse_mini_rule.ml

```diff
@@ -28,6 +28,9 @@ exception InvalidLanguageException of string * string
 exception InvalidPatternException of string * string * string * string
 
 (*e: exception [[Parse_rules.InvalidPatternException]] *)
+
+exception InvalidRegexpException of string * string
```

Comment:
By the way, I don't think I introduced those exceptions, but there's no need to have them suffixed by Exception.
We know it's an exception :)
(It's fine in this PR because it's good to be consistent, but we should fix those later).